rest_framework_extensions/etag/decorators.py

```diff
@@ -134,4 +134,22 @@ def _get_and_log_precondition_failed_response(self, request):
         return Response(status=status.HTTP_412_PRECONDITION_FAILED)
 
 
-etag = ETAGProcessor
\ No newline at end of file
+class APIETAGProcessor(ETAGProcessor):
+    """
+    This class is responsible for calculating the ETag value given (a list of) model instance(s).
+
+    The difference to the base class `ETAGProcessor` is that it cannot be used as `@api_etag()` decorator.
```

Comment:
Yes you are right, this is a relict of the development phase and misleading. I removed that in a new commit.